### PR TITLE
Goap

### PR DESCRIPTION
**Pull request by dnk777**

### CODE REVIEW

**Comment by viciious** [comment]
The check fails for the tutorial gametype

File: gametypes/ctftactics/ctf_players.as [hunk 1]
```diff
@@ -489,6 +489,7 @@ class cPlayer
 
     void refreshMovement()
     {
+		this.ent.aiVisibilityDistance = 999999.9f;
```
Comment (viciious) [inline gametypes/ctftactics/ctf_players.as]: Formatting: tabs vs spaces

File: gametypes/ctftactics/ctf_players.as [hunk 2]
```diff
@@ -527,6 +528,8 @@ class cPlayer
                 this.client.selectWeapon( WEAP_NONE );
                 this.client.pmoveFeatures = this.client.pmoveFeatures & ~PMFEAT_WEAPONSWITCH;
                 this.ent.effects |= EF_PLAYER_HIDENAME;
+				// The player should be visible for bots only on range of 192 units
+				this.ent.aiVisibilityDistance = 192.0f;
```
Comment (viciious) [inline gametypes/ctftactics/ctf_players.as]: Same as above

**Comment by dnk777** [comment]
Its probably because `tutorial.gt` does not include `generic/bots.as` in the script. Wait for a quick fix.

File: gametypes/ctftactics/ctf_players.as (hunk 2 above)
Comment (dnk777) [reply gametypes/ctftactics/ctf_players.as]: I will fix it but tbh the original files were messy at it too... All scripts need a global style fix.

**Pull request merged**